### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Emile59**

### CODE REVIEW

File: src/logger.c [hunk 1]
```diff
@@ -31,6 +35,11 @@ void logger_init(log_level_t log_level)
 		.stopbits = US_MR_NBSTOP_1_BIT
 	};
 	serial_mdw_init_interface(LOGGER_SERIAL_INTERFACE, &serial_option, TIMESTAMP_USED);
+
+	// Initialize UDP client
+	IP4_ADDR(&ipaddr, RPI_IP_ADDR_A, RPI_IP_ADDR_B, RPI_IP_ADDR_C, RPI_IP_ADDR_D);
```
Comment (amof) [inline src/logger.c]: ip address of destination should be a parameter and not fixed

File: src/logger.c [hunk 2]
```diff
@@ -133,22 +142,21 @@ void log_log(log_level_t level, const char *file, uint32_t line, const char *fmt
...
+					serial_mdw_send_bytes(LOGGER_SERIAL_INTERFACE, (uint8_t*)buffer, (uint32_t)length);
+					// Send a return carriage + new line only when using serial link
+					char returnToTheLine_buffer[2] = { '\r', '\n' };
+					serial_mdw_send_bytes(LOGGER_SERIAL_INTERFACE, (uint8_t*)returnToTheLine_buffer, (uint32_t)2);
+					
+					delay_ms(LOGGER_SERIAL_DELAY);
+				}
+				if (logger_log_interface == (log_interface_t)LOG_INTERFACE_ETHERNET || logger_log_interface == (log_interface_t)LOG_INTERFACE_BOTH)
+				{
+					udp_client_send_to((uint8_t*)buffer, (uint32_t)length, &ipaddr, RPI_PORT);
```
Comment (amof) [inline src/logger.c]: the port should not be fixed to RPI_PORT

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+/.vs
```
Comment (amof) [inline .gitignore]: this file should not be commited -> only specific to the way you develop

**Review by amof — CHANGES_REQUESTED** [review]
Few changes before it's good to go !

File: src/logger.c (hunk 2 above)
Comment (Emile59) [reply src/logger.c]: Addition of two functions allowing the user to choose the interface he wants, so the destination port is no longer fixed

File: src/logger.c (hunk 1 above)
Comment (Emile59) [reply src/logger.c]: Same as for the destination port

**Comment by amof** [comment]
Thanks for the addition, made a little rework from your modifications.

**Pull request closed without merging**